### DISCUSSION TITLE
weed mount + tail ?

### DISCUSSION THREAD

**Discussion by prologic** [seaweedfs/seaweedfs#2106] [Q&A]
Hi @chrislusf 👋

Is tailing a file `tail -f` supported with the FUSE support via `weed mount`?

Kind regards

James

**Comment by chrislusf** [comment]
Not sure how it is implemented for local file system.

**Reply by prologic** [reply]
You can see [here](https://unix.stackexchange.com/questions/102905/does-tail-read-the-whole-file) for low-level details on generally how `tail -f` works.

Basically I tried this myself and what I found was that subsequent seek() cand read() calls do not seem to see new data written to the file. I'm not sure why yet... Implementation/design detail of how the Filer and the FUSE layer works?

**Reply by chrislusf** [reply]
The file content are "committed" to filer store when the file is closed. 

Maybe some FUSE interface needs to be implemented.

**Reply by prologic** [reply]
I'll do some more poking around and try to find out...
